syntax = "proto3";

package my;

service MyService {
    rpc RunCommand (MyRequest) returns (MyResponse) {}
}

message MyRequest {
    string name = 1;
}

message MyResponse {
    string message = 1;
}

